test/verify/check-system-info

```diff
@@ -380,5 +381,48 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+class TestPcp(packagelib.PackageCase):
+
+    @skipImage("cockpit-system doesn't have the install-on-demand feature", "rhel-7-5")
+    @skipImage("doesn't have pcp", "debian-stable")
+    @skipImage("https://bugzilla.redhat.com/show_bug.cgi?id=1559414", "rhel-x")
+    def testEnablePcpLink(self):
+        m = self.machine
+        b = self.browser
+
+        # the atomics don't have pcp and can't install additional software
+        if m.atomic_image:
+            self.login_and_go("/system")
+            b.wait_present("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#system-information-enable-pcp")
+            return
+
+        m.execute("pkcon remove -y pcp")
+
+        # the offer to install it should be visible
+        self.login_and_go("/system")
+        b.wait_present("#server-pmlogger-onoff-row")
+        b.wait_not_visible("#server-pmlogger-onoff-row")
+        b.wait_visible("#system-information-enable-pcp")
+        b.wait_in_text("#system-information-enable-pcp", "Enable persistent metrics…")
+        b.click("#system-information-enable-pcp-link")
+        b.wait_present("#cockpit_modal_dialog")
+        b.wait_in_text("#cockpit_modal_dialog .modal-header", "Install Software")
+
+        pmlogger = {
+            "/usr/bin/pmlogger": "",
+            "/etc/systemd/system/pmlogger.service": "[Service]\nExecStart=/usr/sbin/true"
+        }
+
+        self.createPackage("cockpit-pcp", "999", "1", content=pmlogger, postinst="chmod +x /usr/bin/pmlogger; systemctl daemon-reload")
+        self.enableRepo()
+        m.execute("pkcon refresh")
+        m.execute("pkcon install -y cockpit-pcp")
+
+        b.wait_visible("#server-pmlogger-onoff-row")
+        b.wait_not_visible("#system-information-enable-pcp")
```

Comment:
I still don't understand this test case. The above opens the dialog, but never acks the package install, and this now installs the fake package manually. Shouldn't the fake package be created at the top of the test case, and then you let the actual cockpit code install the package all the way through?

Reply:
You're right. I've gone through a lot of changes on this one, and this is a holdout from before I had the mock package. I've changed it to install by clicking the button instead of doing it through `pkcon`. Thanks!